tests/test_workers/test_pkg_manager.py

```diff
@@ -22,7 +22,7 @@
     github.com/kr/pty v1.1.1
     github.com/kr/text v0.1.0
     github.com/op/go-logging v0.0.0-20160315200505-970db520ece7
-    github.com/pkg/errors v0.8.1
+    github.com/pkg/errors v0.8.0 github.com/pkg/errors v0.8.1
```

Comment:
It would be nice to have a test case that verified v0.8.1 is used. Also, to ensure that the second module name is used, maybe change the line to something like:
```
github.com/pkg/ignored v0.8.0 github.com/pkg/errors v0.8.1
```

I don't think we need to do it for every dep here, but a few keys one would be useful.

Reply:
This actually does test that `v0.8.1` is used. I can add a test to for the different name though.